packages/admin/src/components/Board/BoardWrite/style.ts

```diff
@@ -0,0 +1,67 @@
+import { css } from "@emotion/css";
+import { colors } from "@shared/styles/color";
+import { hashClassNames } from "src/lib/hash";
+
+export default () => {
+  const { warn, titleBox, title, errorMessage, horizonLine, submit } =
+    hashClassNames([
+      "warn",
+      "titleBox",
+      "title",
+      "errorMessage",
+      "horizonLine",
+      "submit",
+    ]);
+
+  const main = css`
+    display: flex;
+    flex-direction: column;
+    margin-left: 13rem;
+    padding: 3rem;
+
+    .${warn} {
+      ::-webkit-input-placeholder {
```

Comment:
저희 관리자 페이지는 프리픽서 없이 작업해도 되지 않을까요?
```suggestion
      ::placeholder {
```